lib/opal/nodes/def.rb

```diff
@@ -199,7 +199,10 @@ def compile
               scope.mlhs_mapping[child] = tmp
 
               child.children.each do |child|
-                scope.add_temp variable(child.last)
+                # No support for nested mlhs yet
+                if child.type != :mlhs
+                  scope.add_temp variable(child.last)
+                end
```

Comment:
What do you mean by unsupported arguments? From what I see in grammar.y `mlhs` can include only `norm_arg`, `restarg` or another `mlhs`. All other cases are rejected on the stage of translating to sexp.